requirement.txt

```diff
@@ -0,0 +1,13 @@
+appdirs==1.4.0
```

Comment:
(im nitpicking here) dependency versions are handled by pip
only specifying what libraries are actually needed helps readability and understanding of dependencies of **our** code (i.e. we never referenced decorator or dask)